### ISSUE TITLE
Can't upload rmse,mae and other results to openml server

### ISSUE THREAD

**Issue by SharathKumarReddyAlijarla** [openml/openml-python#1197]
#### Description

Can't upload rmse,mae and other results to openml server
#### Steps/Code to Reproduce

`    root = logging.getLogger()
    root.setLevel(logging.INFO)

    args = parse_args()
    if args.openml_apikey:
        openml.config.apikey = args.openml_apikey
    if args.openml_server:
        openml.config.server = args.openml_server
    else:
        openml.config.server = 'https://test.openml.org/api/v1/'
    clf_pipeline=pipeline()
    run = openml.runs.run_model_on_task(clf_pipeline, args.task_id)
    score = []
    evaluations = run.fold_evaluations['mean_absolute_error'][0]
    for key in evaluations:
        score.append(evaluations[key])
    mae=np.mean(score)
    print(np.mean(score))
    print(np.mean(run.get_metric_fn(sklearn.metrics.mean_squared_error)))
    print(np.mean(run.get_metric_fn(sklearn.metrics.r2_score)))
    logging.info('Task %d - %s; Accuracy: %0.2f' % (args.task_id, task.get_dataset().name, mae))
    
    del_outputdir()
    run.to_filesystem(directory= 'myrun')
    print(args.upload_result)
    
    if args.upload_result:
        run.publish()
        print(run)
        print('Results Uploaded to Openml and stored in local folder myrun')
    else:
        print('Results stored in local folder myrun')`
<!--
Example:
```python
import openml
openml.flows.get_flow(1)
```

```
If the code is too long, feel free to put it in a public gist and link
it in the issue: https://gist.github.com
-->

#### Expected Results
<!-- Example: No error is thrown. Please paste or describe the expected results.-->
want to upload mean absolute error value to openml runs. run is uploaded but this values are not uploaded when run.publish() is executed.I am only able to get results locally,
#### Actual Results
<!-- Please paste or specifically describe the actual output or traceback. -->

#### Versions
<!--
Please run the following snippet and paste the output below.
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import numpy; print("NumPy", numpy.__version__)
import scipy; print("SciPy", scipy.__version__)
import sklearn; print("Scikit-Learn", sklearn.__version__)
import openml; print("OpenML", openml.__version__)
-->


<!-- Thanks for contributing! -->

**Comment by LennartPurucker** [comment]
Solved in #1209
